<launch>
    <arg name="in1" default="/reconstruction/point_clouds/cam1/preprocessed" />
    <arg name="in2" default="/reconstruction/point_clouds/cam2/preprocessed" />
    <arg name="out" default="/reconstruction/point_clouds/fix" />
    <node name="merging_fix" pkg="registration" type="merging_node" args="fix $(arg in1) $(arg in2) $(arg out)" output="screen" />
    <node name="preprocessing_fix" pkg="registration" type="preprocessing_node" args="fix $(arg out)" output="screen" />
</launch>